Makefile

```diff
@@ -63,24 +63,33 @@ lint:
 	@echo "------------------"
 	@go vet $(PACKAGES)
 
-.PHONY: build  
+.PHONY: build
 build:
 	@echo "------------------"
 	@echo "--> Build Chaos Operator"
 	@echo "------------------"
-	@go build -o ${GOPATH}/src/github.com/litmuschaos/chaos-operator/build/_output/bin/chaos-operator -gcflags all=-trimpath=${GOPATH} -asmflags all=-trimpath=${GOPATH} github.com/litmuschaos/chaos-operator/cmd/manager 
+	@go build -o ${GOPATH}/src/github.com/litmuschaos/chaos-operator/build/_output/bin/chaos-operator -gcflags all=-trimpath=${GOPATH} -asmflags all=-trimpath=${GOPATH} github.com/litmuschaos/chaos-operator/cmd/manager
 
 .PHONY: test
 test:
 	@echo "------------------"
 	@echo "--> Run Go Test"
 	@echo "------------------"
-	@go test ./... -coverprofile=coverage.txt -covermode=atomic -v 
+	@go test ./... -coverprofile=coverage.txt -covermode=atomic -v
 
-.PHONY: dockerops 
-dockerops: 
+.PHONY: dockerops
+dockerops:
 	@echo "------------------"
-	@echo "--> Build & Push chaos-operator docker image" 
+	@echo "--> Build & Push chaos-operator docker image"
 	@echo "------------------"
 	sudo docker build . -f build/Dockerfile -t $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG)
-	REPONAME=$(DOCKER_REPO) IMGNAME=$(DOCKER_IMAGE) IMGTAG=$(DOCKER_TAG) ./buildscripts/push
\ No newline at end of file
+	REPONAME=$(DOCKER_REPO) IMGNAME=$(DOCKER_IMAGE) IMGTAG=$(DOCKER_TAG) ./buildscripts/push
+
```

Comment:
```suggestion
.PHONY: unused-package-check
```

Reply:
unused-package-check call from `deps` in line no. 27

Reply:
What if someone wants to call it without calling `deps`?

Reply:
You can call directly `make unuse-package-check`

Reply:
Good to go then